beets/autotag/__init__.py

```diff
@@ -22,7 +22,13 @@
 from beets import config
 
 # Parts of external interface.
-from .hooks import AlbumInfo, TrackInfo, AlbumMatch, TrackMatch  # noqa
+from .hooks import (  # noqa
+    AlbumInfo,
+    TrackInfo,
+    AlbumMatch,
+    TrackMatch,
+    Distance,
+)
```

Comment:
I think this change can be reverted now because `Distance` is no longer used here.

Reply:
Oops; I'm obviously wrong about that. Not changing anything!